styles.css

```diff
@@ -22,19 +59,68 @@ body {
     display: none;
   }
 
-  body {
-    background-color: #e5e5e5;
-    margin: 0;
-    padding: 0;
+  .desktop-popup {
+    position: fixed;
+    top: 0;
+    margin: 5%;
+    background-color: #fff;
+    z-index: 5;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    height: 100vh;
+    border-radius: 16px;
+    padding: 2%;
+    -webkit-backdrop-filter: blur(10px);
   }
 
-  img {
-    transition: transform 1s ease-in-out;
-    transition: all 0.3s;
+  .desktop-popup-img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .popup-content-box {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+  }
+
+  .lang-button-holder {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .hide-desktop-popup {
+    display: none;
+  }
+
+  .popup-buttondiv {
+    display: flex;
+    justify-content: center;
   }
 
-  img:hover {
-    transform: scale(1.1);
+  .desktoppopup-card {
+    display: flex;
+    flex-direction: column;
+    padding: 0%;
+    height: 100%;
+    width: 100%;
+    overflow-y: scroll;
+  }
+
+  .popup-close {
+    position: absolute;
+    top: 3%;
+    right: 5%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: #67798e;
+  }
```

Comment:
- [OPTIONAL] Kindly consider adding this style rule `cursor: pointer;` to this selector. It would make the UI more interactive for users. 👍 😎